CHANGES

```diff
@@ -2,23 +2,16 @@
                                Changes in 4.0
 ===============================================================================
 
-# Implemented basic MPI-4 session API using MPIX interface.
+# All MPI-4 APIs have been implemented. Major MPI-4 features include MPI
+  sessions, partitioned point-to-point communications, events in the MPI tool
+  information interface, large-count functions, persistent collectives,
+  MPI_Comm_idup_with_info, MPI_Isendrecv and MPI_Isendrecv_replace,
+  MPI_Info_get_string, MPI_Comm_split_type with new split_type --
+  MPI_COMM_TYPE_HW_GUIDED and MPI_COMM_TYPE_HW_UNGUIDED.
 
-# Implemented basic MPI-4 partitioned communication API using MPIX interface.
+# Add QMPI support.
```

Comment:
Might want to note that this is an experimental feature.

Reply:
Changed to `Add QMPI (experimental) support.`